UI/index.erb

```diff
@@ -21,8 +21,8 @@
                         <h1 class="navbar-brand"><span class="sr-only"><%= @strings['applicationTitle'] %></span></h1>
                     </div>
                     <div class="nav navbar-nav navbar-right loginContainer col-lg-8 col-md-8 col-sm-8 col-xs-8">
-                        <a class="btn btn-link" href="https://ctools.umich.edu/portal/login" id="ctoolsLogin">CTools<span class="visible-lg-inline visible-md-inline visible-sm-inline"> Login</span></a>
-                        <a class="btn btn-link" href="https://umich.instructure.com" id="canvasLogin">Canvas<span class="visible-lg-inline visible-md-inline visible-sm-inline"> Login</span></a>
+                        <a class="btn btn-default" href="https://ctools.umich.edu/portal/login" id="ctoolsLogin">CTools<span class="visible-lg-inline visible-md-inline visible-sm-inline"> Login <strong>»</strong></span></a>
```

Comment:
Should the URLs be properties?  Then test instances won't have to point to production instances.